tests/e2e/functional_test.go

```diff
@@ -524,6 +541,50 @@ var _ = Describe("Functional e2e", Serial, func() {
 
 	})
 
+	It("Should update the child ISBService with an excluded field", func() {
+
+		// new ISBService spec
+		rawSpec, err := json.Marshal(ISBServiceSpecExcludedField)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		updateISBServiceRolloutInK8S(isbServiceRolloutName, func(rollout apiv1.ISBServiceRollout) (apiv1.ISBServiceRollout, error) {
+			rollout.Spec.InterStepBufferService.Spec.Raw = rawSpec
+			return rollout, nil
+		})
+
+		if dataLossPrevention == "true" {
```

Comment:
nit: I believe the block of code should apply whether or not `dataLossPrevention == true`?